### PR TITLE
Testing : upadted file 6

### PR DESCRIPTION
**Pull request by shiva-024**
upadted file 6

### CODE REVIEW

**Comment by vaalu** [comment]
Can you add a line for unit testing.

File: filep1.txt [hunk 1]
```diff
@@ -1 +1,3 @@
 GitHub flow training!
+
+Updated......
```
Comment (vaalu) [inline filep1.txt:3]: A simple comment

**Review by shiva-024 — COMMENTED** [review]
Done

**Review by vaalu — APPROVED** [review]
Comments addressed

File: filep1.txt (hunk 1 above)
Comment (Poojashreeg) [reply filep1.txt:3]: Done.

**Pull request merged**